Comparison/src/cz/fidentis/comparison/procrustes/GPA.java

```diff
@@ -178,6 +178,10 @@ public void superimpose(){
         
         for(int i = 0; i < configs.size(); i++){
             ICPTransformation tran = configs.get(i).rotate(oldMean);  
+            
+            if(tran == null)

```

Comment:
Za akych okolnosti sa toto moze stat? Nesposobi to problem ak tato metoda vrati `null`? (EDIT: vidim ze to riesis nizsie)
Otazka je ci sa da tato situacia riesit lepsie a ci nebude lepsie napr. vratit prazdny list namiesto nullu alebo tak